api/tests/test_agents_service.py

```diff
@@ -27,6 +27,7 @@ def mock_functions(self, mocker):
         mocker.patch("agents_service.TrainingDataRepository.__init__", lambda x: None)
         mocker.patch("agents_service.UsersRepository.__init__", lambda x: None)
         mocker.patch("agents_service.UsersInputsRepository.__init__", lambda x: None)
+        mocker.patch("agents_service.AnalyticsRepository.__init__", lambda x: None)
```

Comment:
create sepate file for test_analytics_service

Reply:
Done